models/llama-2-7b-vllm/predict.py

```diff
@@ -0,0 +1,151 @@
+import asyncio
+import functools
+import io
+import time
+import zipfile
+from typing import Optional, Tuple
+
+import torch
+from cog import BasePredictor, ConcatenateIterator, Input, Path
+
+from config import (LOCAL_DEFAULT_INFERENCE_WEIGHTS_PATH,
+                    REMOTE_DEFAULT_INFERENCE_FILES_TO_DOWNLOAD,
+                    REMOTE_DEFAULT_INFERENCE_WEIGHTS_PATH)
+from src.download import Downloader
+from src.inference_engines.vllm_engine import vLLMEngine
+from src.utils import maybe_download_with_pget
+
+# This prompt formatting was copied from the original CodeLlama repo:
+# https://github.com/facebookresearch/llama/blob/6c7fe276574e78057f917549435a2554000a876d/llama/generation.py#L44
```

Comment:
this can probably be removed, looks like it was a comment pulled in from a different branch

Reply:
Done!